### ISSUE TITLE
Fix pandas issue

### ISSUE THREAD

**Issue by hmedal** [hmedal/LANS#15]
This issue needs to be fixed in versions 5 an 6.1.

The first issue was noticed with pandas (same issues on both versions of LANS). The version of pandas that was available on the system was 0.21.1. None of these issues are seen when using pandas version 0.19.1
In the file role_mining.py, an error occurs at line:
    feature_data = pd.read_csv(feature_file,delimiter=',',usecols=[0,1,2,3,4,5,6])
    features = feature_data[[1,2,3,4,5,6]].as_matrix()
 
What I did to work around it before rolling back to pandas version 0.19.1 was to specify those columns, which made it past that error.

**Comment by sarahharun** [comment]
Dr Medal, I almost forgot everything. In order to fix it, I need the source code of version 5 and 6.1 and the error message.

**Comment by hmedal** [comment]
The source code is in this Github repo, right? And the error message is listed in Mandy's comment above, right?

**Comment by sarahharun** [comment]
Yes, sorry I lost the github repo link for version 5 and 6, but now I got them from the branches. The sponsors are saying that an error occurred in those two lines, but I need to know what was the error (the description of the error e.g. column missing, argument not matching etc). I will check the version of Pandas at Shadow, if it is not 0.21.1, then I can try to reproduce the error in my laptop. But without knowing the exact error, I may never be sure whether I am addressing the correct error.

**Comment by sarahharun** [comment]
Pandas version is 0.16.2 at Shadow.

**Comment by hmedal** [comment]
Ok I will ask the sponsor for the issue. And please test it with pandas 0.19.1 on Shadow.
